mysql-test/suite/encryption/t/innochecksum.test

```diff
@@ -300,4 +300,14 @@ if (0 && $have_debug) { # these messages sometimes fail to appear
 --move_file $MYSQLD_DATADIR/test/t6.ibd.backup $MYSQLD_DATADIR/test/t6.ibd
 
 --source include/start_mysqld.inc
+
+--echo # Trigger AHI and test pages_hashed, pages_old as non zero from sys view
+
+SELECT STRAIGHT_JOIN COUNT(t6a.a)
+FROM t6 AS t6a, t6 AS t6b
+WHERE t6a.a = t6b.a;
+
+SELECT SUM(pages_hashed)>0 `1`, SUM(pages_old)>0 `1`
+FROM sys.innodb_buffer_stats_by_table;
```

Comment:
This would fail if the server had been built with `cmake -DPLUGIN_PERFSCHEMA=NO`. I fixed this up in dfb22d9e4fe192cb99ccbbc38add888fc36c9e45.